nametoaddr.c

```diff
@@ -126,7 +126,14 @@ pcap_nametoaddr(const char *name)
 	bpf_u_int32 **p;
 	struct hostent *hp;
 
+#ifdef _WIN32
+#pragma warning (push)
+#pragma warning (disable: 4996) /* disable MSVC's gethostbyname() warning here */
+	if ((hp = gethostbyname(name)) != NULL) {
+#pragma warning (pop)
```

Comment:
Wouldn't it be better to add a `#define _WINSOCK_DEPRECATED_NO_WARNINGS` (or in `CLFAGS`) instead of "*polluting*" the code with these pragmas? Besides these pragmas should be inside an `#ifdef _MSC_VER`.

Similarily for `GetVersion()` where one can define `BUILD_WINDOWS.